app/controllers/reading_list_items_controller.rb

```diff
@@ -1,6 +1,8 @@
 class ReadingListItemsController < ApplicationController
   def index
     @reading_list_items_index = true
+    @reading_collections = ReadingCollection.where(user_id: session_current_user_id).to_a
+    # Article.joins(:taggings).joins(:tags).select("articles.*").where("tags.name = 'javascript'")
```

Comment:
Is this laying out functionality to generate a temporary collection for testing, or pseudocode of how we might generate article per collection later?

Reply:
Not applicable for this moment, just a note to touch on in standup.

Reply:
Yup, it's more like a "note" for later when we add the ability to find articles of a certain tag.